src/sinol_make/util.py

```diff
@@ -108,6 +108,43 @@ def check(path):
 		return None
 
 
+def save_config(config):
+	"""
+	Function to save nicely formated config.yml
+	"""
+
+	# We add the fields in the config.yml in a particular order to make the config more readable
+	# The fields that are not in this list will be appended to the end of the file
```

Comment:
Now that I think about it, what is a reasonable scenario where a user would have custom fields in the config? How would such fields be used? Maybe we should give a warning or an error in case of non-supported fields? I'm open for discussion.

Reply:
I've seen `oiejq_path` in some tasks (it was used by the `runner.py`), but other than that I don't recall seeing anything.

Reply:
I leave the decision up to you.

Reply:
I think we should leave this as is

Reply:
Sure, we can leave it. Are there any reasons for why you think it's the best choice?

Reply:
I think that the `config.yml` should still be compatible with other scripts (`runner.py` for example). But I guess a warning would be nice.

Reply:
Good point!

Reply:
Re-add me as a reviewer when you'll finish this PR (e.g. after adding the warning).

Reply:
I added the warning: bab9b9b933e554db95061ea24205fbda93df1884